app/main.ipynb

```diff
@@ -192,18 +249,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=1)",
```

Comment:
In Task 5.1, the mean should be calculated along the columns (axis=0) to find the mean for each column, not along the rows (axis=1).